src/controllers/queries/__tests__/index.test.ts

```diff
@@ -189,23 +188,12 @@ describe('QueriesController', () => {
           companyId: '2345789',
           action: 'success',
           totalScope3CarbonEmissions: '25',
-          partialResponse: 1,
         })
         .then(toHTMLString)
 
       expect(result).to.equal('scope3_error_scope3')
     })
   })
-  describe('viewing query responses', () => {
```

Comment:
was this a duplicate of something?

Reply:
looks like, it was me rebasing with out branch (probably)